.changesets/feat_geal_async_coprocessor.md

```diff
@@ -0,0 +1,5 @@
+### asynchronous coprocessor execution ([Issue #3297](https://github.com/apollographql/router/issues/3297))
+
+This adds a new `asynchronous` option at all coprocessor stages to allow the router to continue handling the request or response without waiting for the coprocessor to respond. This implies that the coprocessor response will not be used. This is targeted at use cases like logging and auditing, which do not need to block the router
```

Comment:
Could you add a full example of configuration with this new option please ?

Reply:
fixed in [db39565](https://github.com/apollographql/router/pull/4902/commits/db395654b60678a5348d065b56615d86c9c0e41c)